### PR TITLE
Updating docs for release

### PR DESCRIPTION
**Pull request by jessicafalk**

### CODE REVIEW

File: docs/fragments/machine-setup.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ Once you have navigated to the **Workloads** tab:
   - The SpatialOS GDK for Unity requires **Unity 2019.1.3**, which you already installed in step 1.
   - Make sure **Visual Studio Tools for Unity** is included (there should be a tick next to it).
 
-> **Warning**: Older versions of Visual Studio 2017 have been known to cause some issues with Unity 2018.3.11 - the issues are projects loading and unloading frequently, and Intellisense breaking. If you do experience these issues, try updating to the latest version of Visual Studio 2017.
+> **Warning**: Older versions of Visual Studio 2017 have been known to cause some issues with Unity 2019.1.3 - the issues are projects loading and unloading frequently, and Intellisense breaking. If you do experience these issues, try updating to the latest version of Visual Studio 2017.
```
Comment (paulbalaji) [inline docs/fragments/machine-setup.md]: `try updating to the latest version of Visual Studio 2017.` or just `try updating to the latest version of Visual Studio`?

File: docs/modules/mobile/prepare-project.md [hunk 2]
```diff
@@ -27,9 +27,9 @@ To connect to a cloud deployment using your mobile device, you need to use the [
 1. In the **Dev Auth Token Settings** specify the lifetime of the token and the path to the [Resources folder](https://unity3d.com/learn/tutorials/topics/best-practices/resources-folder) that you would like to store the generated token in.
 
     > Your token expires after the amount of days that you specified in the configuration window. Regenerate the token whenever that happens.
-
+1. (for iOS) Ensure that **Save token to file** is checked.
 1. Select **Save** and close the window.
-1. Select **SpatialOS** > **Generate Dev Authentication Token**. This generates a `DevAuthToken.txt` asset in the folder you specified in the configuration window. This token is used to authenticate when connecting to a cloud deployment.
+1. Select **SpatialOS** > **Generate Dev Authentication Token**. This stores the dev auth token inside the Unity Player Preferences. If **Save token to file** inside the configuration window is checked, it will also generate a `DevAuthToken.txt` asset in the folder you specified in the configuration window. This token is used to authenticate when connecting to a cloud deployment.
```
Comment (paulbalaji) [inline docs/modules/mobile/prepare-project.md]: actually it's **SpOS** > **Dev Authentication Token** > **Generate Token** now 😬

Comment (paulbalaji) [reply docs/modules/mobile/prepare-project.md]: should probably update this if there are other places we tell people to generate a token

File: docs/fragments/machine-setup.md (hunk 1 above)
Comment (jessicafalk) [reply docs/fragments/machine-setup.md]: have we tested Visual Studio 2019?

File: docs/modules/mobile/prepare-project.md (hunk 2 above)
Comment (jessicafalk) [reply docs/modules/mobile/prepare-project.md]: seems to be the only place

**Pull request merged**